src/lib.js

```diff
@@ -63,17 +63,27 @@ async function start(fields) {
   log('info', 'Retrieve all informations for each bank accounts found')
 
   const today = moment().format('YYYY-MM-DD')
-  const lastYear = moment()
-    .subtract(1, 'years')
+  const tenYearsAgo = moment()
+    .subtract(10, 'years')
```

Comment:
10 years is the maximum?

**For the futher**: it may be interesting to detect if there are already operations for this bank account. In this case, look for one year (more efficient).

Reply:
I don't know, it's just that one year seemed far too short, and ten years works :man_shrugging: 

And I don't know if it would be much more efficient to fetch only one year if there are already existing operations: Cozy already implements a filter/resolver in the BankingKonnector.

Reply:
Indeed, Cozy implements a filter in the bankingKonnector, but it must process all operations to detect the new and the operations already saved, each hour... The idea is to avoid doing too much unnecessary operations (code). So, improve the runtime _(reduce CPU and RAM consumption, ...)_

It's just an idea :)